### ISSUE TITLE
SerializationError: failed to decode S3 XML error response

### ISSUE THREAD

**Issue by lowstz** [juicedata/juicesync#6]
``` bash
./juicesync qiniu://KEY:SECRET@test-bucket.cn-east-1-s3.qiniu.com/  qingstor://KEY:SECRET@other-test-bucket.sh1a.qingstor.com/
```

Output log:
```
2018/11/04 11:17:01.184505 <WARNING>: Failed to replicate testdir/testfile from qiniu://test-bucket to qingstor://other-test-bucket: SerializationError: failed to decode S3 XML error response
        status code: 404, request id: , host id:
caused by: expected element type <Error> but have <html>
```

**Comment by davies** [comment]
Looks like Qiniu S3 API is not compatible with the S3 SDK which expect a XML error response rather than the actual HTML response.

Have told Qiniu team to look into this.

Btw, this should not block anyone, since the object doest not exits.

**Comment by davies** [comment]
There is something wrong with api-s3.qiniu.com, it's good after switching to cn-east-1-s3.qiniu.com

Fixed by d0b93b7.
